tests/js/spec/components/avatar.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import Avatar from 'app/components/avatar';
```

Comment:
To clarify – you don't have to change these to use Enzyme. Just some thoughts for the future.